src/impl_/extract_argument.rs

```diff
@@ -255,57 +257,29 @@ impl FunctionDescription {
             num_positional_parameters + self.keyword_only_parameters.len()
         );
 
-        let varargs = if positional_args_provided > num_positional_parameters {
-            let (positional_parameters, varargs) = args_slice.split_at(num_positional_parameters);
-            output[..num_positional_parameters].copy_from_slice(positional_parameters);
-            V::handle_varargs_fastcall(py, varargs, self)?
-        } else {
-            output[..positional_args_provided].copy_from_slice(args_slice);
-            V::handle_varargs_fastcall(py, &[], self)?
-        };
+        // Handle positional arguments
+        let (positional_parameters, remaining) =
+            if positional_args_provided > num_positional_parameters {
```

Comment:
If we are at code-golfing this, wouldn't

```rust
args_slice.split_at(positional_args_provided.min(num_positional_parameters))
```

suffice?

Reply:
Nice! I force-pushed to also move some other code down to make this even tighter :)